view/loadView.js

```diff
@@ -215,13 +215,7 @@ module.exports = function loadView(app, passport) {
     (_req, _res) => {
       if (_req.user && _req.user.name) {
         const token = jwtUtil.createToken(_req.user.name, _req.user.name);
-        _res.cookie(
-          'Authorization',
-          token, {
-            secure: true,
-            httpOnly: true,
-          }
-        )
+        _res.cookie('Authorization', token);
```

Comment:
Removing {secure:true, httpOnly: true} (was added in #140 ) because token is not set in cookie and hence SSO user gets Forbidden when USE_ACCESS_TOKEN=true.
@annyhe Please confirm if this is okay.